src/main/java/org/mitre/synthea/export/FhirR4.java

```diff
@@ -1675,18 +1675,20 @@ private static BundleEntryComponent allergy(RandomNumberGenerator rand,
             new AllergyIntolerance.AllergyIntoleranceReactionComponent();
         reactionComponent.addManifestation(mapCodeToCodeableConcept(manifestation, SNOMED_URI));
         HealthRecord.ReactionSeverity severity = allergy.reactions.get(manifestation);
-        switch (severity) {
-          case MILD:
-            reactionComponent.setSeverity(AllergyIntolerance.AllergyIntoleranceSeverity.MILD);
-            break;
-          case MODERATE:
-            reactionComponent.setSeverity(AllergyIntolerance.AllergyIntoleranceSeverity.MODERATE);
-            break;
-          case SEVERE:
-            reactionComponent.setSeverity(AllergyIntolerance.AllergyIntoleranceSeverity.SEVERE);
-            break;
-          default:
-            // do nothing
+        if (severity != null) {
```

Comment:
Adding this `null` check makes sense... but do you know which modules were creating allergies without severities? Maybe we should fix those.

Reply:
I don't *think* there are any, this was flagged by the sonatype-lift bot so I fixed it preemptively.